modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -213,6 +218,15 @@ def perform_pdf_upload(location, file_path, metadata, form)
         lighthouse_service.perform_upload(**upload_params)
       end
 
+      def upload_pdf_to_s3(id, file_path, metadata, submission, form)
+        config = SimpleFormsApi::FormRemediation::Configuration::VffConfig.new
+        attachments = get_form_id == 'vba_20_10207' ? form.attachment_guids : []
+        s3_client = config.s3_client.new(
+          config:, type: :submission, id:, submission:, attachments:, file_path:, metadata:
+        )
+        s3_client.upload
```

Comment:
do we want an error for this as well for s3 upload failure?

Reply:
Error handling is happening under the hood with AWS S3 Client and CarrierWave